spec/index.bs

```diff
@@ -4039,9 +4039,14 @@ Issue(gpuweb/gpuweb#354): Finish defining multithreading API and add `[Serializa
 ### Shader Module Creation ### {#shader-module-creation}
 
 <script type=idl>
+dictionary GPUShaderModuleCompilationHints {
```

Comment:
Oh, one more thought.

nit: I think this should be singular as it's one entry in a `record<>` of hints.